project-koku/koku
# [COST-4975] - Fixing GCP summary status for sources endpoint

## Jira Ticket

[COST-4975](https://issues.redhat.com/browse/COST-4975)

## Description

This change will change the ordering for our manifest query to descending, in order to get the latest created manifest returned rather than the oldest. Most cloud providers have a single manifest per month but GCP has multiples so we need to make sure we are selecting the latest.

## Testing

1. Checkout main
2. Restart Koku
3. Create a REAL GCP source (can still use nise data but needs to use the real flow)
4. Wait for data to ingest
5. head to the sources endpoint. Example /api/cost-management/v1/sources/2/
6. see summary status shows pending
7. Checkout this branch
8. Hit the above endpoint again (Might need to clear cache)
9. See we no show correct timestamps along side complete state.

## Release Notes

```markdown
* [COST-4975](https://issues.redhat.com/browse/COST-4975) Fix source status for GCP summary states
```
